src/main/java/org/fogbowcloud/manager/core/rest/controllers/ComputeOrdersController.java

```diff
@@ -57,7 +59,9 @@
 	}
 
 	@RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
-	public ResponseEntity<Boolean> deleteCompute(@PathVariable String id) {
+	public ResponseEntity<Boolean> deleteCompute(@PathVariable String id,
+			@RequestHeader(value = "accessId") String accessId) throws UnauthorizedException, OrderManagementException {
```

Comment:
Rather than use `"accessId"` you can use the constant `ACCESS_ID_HEADER_KEY`. Why `@delete` does not have the `localTokenId` as a header?